sample-scenarios/sample1/README.md

```diff
@@ -112,7 +112,7 @@ Below two screenshots shows that both John and TOm can subscribe to Employee_inf
 
 ### References
 
-[1] -[ https://docs.wso2.com/display/AM210/Key+Concepts#KeyConcepts-APIvisibilityandsubscription](https://docs.wso2.com/display/AM210/Key+Concepts#KeyConcepts-APIvisibilityandsubscription)
+[1] -[https://docs.wso2.com/display/AM220/Key+Concepts#KeyConcepts-APIvisibilityandsubscription](https://docs.wso2.com/display/AM220/Key+Concepts#KeyConcepts-APIvisibilityandsubscription)
```

Comment:
This should be "AM250", isn't it?

Reply:
Changed applicable instances found in the sample scenario README files.